cli/endpoints/online/profiling/profiling_job_tmpl.yml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+command: >
+  profiling.sh -p {inputs.payload}
```

Comment:
where is profiling.sh?

Reply:
in the image

Reply:
Base images should never expose implementation details like the name of the script being invoked.

Instead, the base image should use `profiling.sh` in the default command, so that we can swap it out with, e.g., a Python or Rust script, in the future.